electron/js/menu/developer.js

```diff
@@ -42,32 +43,69 @@ var devToolsTemplate = {
 
 var devProductionTemplate = {
   label: 'Production',
-  click: function() {getPrimaryWindow().loadURL(config.PRODUCTION_URL);},
+  type: 'radio',
+  checked: env === 'production',
```

Comment:
Instead of using fixed strings here, we should use constants. Do we have `'production'` defined somewhere already?